### ISSUE TITLE
Can I move cursor by mouse clicking?

### ISSUE THREAD

**Issue by x10000year** [kingToolbox/WindTerm#239]
xshell supports move cursor to the location of mouse by ctrl+click. can windterm supports the same thing?

**Comment by MostHated** [comment]
This sounds great. 

I am assuming that it would it be possible to have/store a list of commonly used snippets that require no modification that can just be sent as is? Then, with the configuration of the snippets, are they all just available globally, or can it be associated with specific stored connections? Ex. If on Server-A I always need to run commands x,y, and z pretty often, but I use different commands on Server-2, it could have a separate list of commands, but then there is a global one they both can use? Being able to store them and just have them sent to the session, either by a dropdown list, hotkey, completion, select them from a sidebar and double click it to send it to the session "as-is" would be awesome (any of those methods, not necessarily any specific one). 

Overall, this sounds like an amazing addition. :+1:

**Comment by kingToolbox** [comment]
@MostHated 
Yes, if you do not specify any `$(...)`, then the snippet will be sent directly as is.

If you notice, every session has a `system` attribute, like `linux`, `cmd`, `powershell` and so on. Different `systems` have their own `command snippets`. Sessions that use the same `system` share all `command snipptes` of that `system`. Therefore, all `Cmd` sessions share the `Cmd`'s command snipptes, all `Powershell` sessions share the `Powershell`'s command snippets. The command snipptes of different systems are independent.  

If you want a `command snippte` to be used for both `cmd` and `powershell`, then you can define it as: 

> {
>   "snippet" : "...",
>   "systems" : ["cmd", "powershell"],
>   ...
> }

If you want one abc to be used for all `systems`, then you can define it as: 

> {
>   "snippet" : "...",
>   "systems" : [ "*" ],
>   ...
> }

The specific format has not been finalized, but it is probably the case. You are welcome to make any suggestions and we can discuss them together.

**Comment by kingToolbox** [comment]
Hello, I'm glad to have the opportunity to inform you that the feature you want is finally implemented. You can download [WindTerm_2.4.0_Prerelease_2](https://github.com/kingToolbox/WindTerm/releases/tag/2.4-prerelease) and check it. If there still have any needs, please let me know, thank you.

You can read [Free Type Mode](https://kingtoolbox.github.io/2022/04/12/free_type_mode/) to learn more.
